src/pages/holofuel/Inbox/Inbox.test.js

```diff
@@ -0,0 +1,179 @@
+import React from 'react'
+import { render, fireEvent, act, within } from '@testing-library/react'
+import wait from 'waait'
+import { ApolloProvider } from '@apollo/react-hooks'
+import { MockedProvider } from '@apollo/react-testing'
+import apolloClient from 'apolloClient'
+import Inbox, { TransactionRow } from './Inbox'
+import { pendingList } from 'mock-dnas/holofuel'
+import { TYPE } from 'models/Transaction'
+import HolofuelOfferMutation from 'graphql/HolofuelOfferMutation.gql'
+import HolofuelAcceptOfferMutation from 'graphql/HolofuelAcceptOfferMutation.gql'
+import HolofuelActionableTransactionsQuery from 'graphql/HolofuelActionableTransactionsQuery.gql'
+
+const actionableTransactions = pendingList.requests.concat(pendingList.promises).reverse().map(item => {
+  if (item.event[2].Request) {
+    return item.event[2].Request
+  } else if (item.event[2].Promise) {
+    return item.event[2].Promise.tx
+  } else {
+    throw new Error('unrecognized transaction type', item.toString())
+  }
+})
+
+jest.mock('data-interfaces/EnvoyInterface')
+// mocking Header because it depends on Router
+jest.mock('components/holofuel/Header')
+
+describe('Inbox Connected', () => {
+  it('renders', async () => {
+    let getAllByRole
+
+    await act(async () => {
+      ({ getAllByRole } = render(<ApolloProvider client={apolloClient}>
+        <Inbox />
+      </ApolloProvider>))
+      await wait(15)
+    })
+
+    const listItems = getAllByRole('listitem')
```

Comment:
@JettTech - hey, this is potentially a little bit more elegant way of doing some of what we were doing in those other tests. Rather than using a `data-testid`, we use an aria role, which means we get a bit more accessibility and the same pattern for testing.